doc/content/modules/user-manual/pages/release-notes/2025.4.0.adoc

```diff
@@ -86,6 +86,25 @@ action def ActionDef1 {
 ```
 - Improve `PortDefinition` textual import by creating the required `ConjugatedPort` during the import process.
 
+- Improve `TransitionUsage` import with _implit_ source. In the following example, the source of outgoing `TransitionUsages` of the `DecisionNode` are now properly resolved.
```

Comment:
- _implit_ -> _implicit_
- Only one sentence per line please.
- 
Could you please take the time to review your contributions, especially the sentences in the changelog and the release note please? It will help you finding the mistakes and fix them.

I will handle this one.